extras/ci/analytics/ci_health.py

```
@@ -656,6 +751,46 @@ def generate_health_html(queue_data, failures, output_dir):
     else:
         failures_html = "<p>No recent CI failures.</p>"
 
+    # Merge queue section
+    mq_html = ""
+    if mq_data:
+        summary = mq_data.get("summary", {})
+        total_checks = summary.get("success", 0) + summary.get("failure", 0) + summary.get("cancelled", 0) + summary.get("in_progress", 0)
```

Ruff (0.15.4)

[error] Local variable `total_checks` is assigned to but never used

Remove assignment to unused variable `total_checks`

(F841)